### ISSUE TITLE
Batch operations on search results

### ISSUE THREAD

**Issue by FelixZhang** [cnwangjie/better-onetab#331]
Please consider adding batch operations e.g. "Move into new group" "Open in new window" "Delete" on the search results within better-onetab settings, it will be a killer feature from my POV.

**Comment by iProgramStuff** [comment]
I agree with this. Being able to work with search results directly would massively enhance this extension.

I really don't like how there is no one to open tabs from search results directly and clicking them just brings them to their spot in the full list. I feel like search results should almost be treated as their own list with most of the controls and options of a regular list - but actions would apply on the list they are from (i,.e. deleting them).
